added sketch image converter

deepak-chouhan opened the pull request:
# Description

Please include a general summary of the change and/or which issue it fixed. 

Fixes #115

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] New Script (non-breaking change which adds a new script to the repository)
- [ ] This change requires a documentation update

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Screenshot (if applicable)
### Before
![](https://raw.githubusercontent.com/deepak-chouhan/General-Purpose-Scripts/main/scripts/image_converter/example.jpg)

### After
![](https://raw.githubusercontent.com/deepak-chouhan/General-Purpose-Scripts/main/scripts/image_converter/example_converted.jpg)

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

OBITORASU commented:
Add your script to main readme please.

deepak-chouhan commented:
> Add your script to main readme please.

okay

deepak-chouhan commented:
@OBITORASU I have update the added the script to readme

OBITORASU commented on scripts/image_converter/requirements.md:
```diff
@@ -0,0 +1,5 @@
+# Sketch image converter
```
This file should be named readme.md
Make a seperate file called requirements.txt for requirements.

deepak-chouhan replied:
done

deepak-chouhan commented:
@OBITORASU , I have added separate requirements.txt file and also renamed the requirements.md to readme.md. please check if there is anything I need to do.

OBITORASU commented:
Ty merged!
